src/models/Call.ts

```diff
@@ -637,6 +643,7 @@ export class ElementCall extends Call {
             baseUrl: client.baseUrl,
             lang: getCurrentLanguage().replace("_", "-"),
             fontScale: `${SettingsStore.getValue("baseFontSize") / FontWatcher.DEFAULT_SIZE}`,
+            analyticsID,
```

Comment:
This could do with a comment explaining that we trust EC with the analytics ID, as discussed internally

Reply:
As we discussed. Thanks for reminding me.

Reply:
Comment was added